sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -628,6 +628,8 @@ Table getTable(
           client
               .tables()
               .get(ref.getProjectId(), ref.getDatasetId(), ref.getTableId())
+              // avoid retrieving the STORAGE_STATS from the table (which is the default option)
+              .set("view", "BASIC")
```

Comment:
This would be breaking change for anyone using the extra metadata coming from STORAGE_STATS. We should add a new flag/option instead and include a warning about our plan to change the default mode.

Reply:
changed the impl to address this